src/server/server.ts

```diff
@@ -60,6 +61,10 @@ async function serveArticlePost(
 
 async function serveArticle(req: Request, res: ExpressResponse): Promise<void> {
   try {
+    // mock content from mapi
+    if (req.query.date || req.query.filter) {
```

Comment:
What's this checking for?

Reply:
Updated the comment. In the context of a webview it will request <mapi_url>?date=x&filter=newer to fetch more blocks. But we can mock that or make a request to mapi for local development.